cmake/sysbuild/suit.cmake

```diff
@@ -293,6 +309,23 @@ function(suit_create_package)
     suit_create_cache_partition("${CACHE_CREATE_ARGS}")
   endforeach()
 
+  suit_get_manifest(${SB_CONFIG_SUIT_ENVELOPE_APP_RECOVERY_TEMPLATE_FILENAME} INPUT_APP_RECOVERY_ENVELOPE_JINJA_FILE)
+  message(STATUS "Found app_recovery manifest template: ${INPUT_APP_RECOVERY_ENVELOPE_JINJA_FILE}")
```

Comment:
Please move after the if that checks if not empty

Reply:
Done